sentinel-dashboard/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM amd64/buildpack-deps:buster-curl as installer
+
+ARG SENTINEL_VERSION
```

Comment:
it's good to give a default value is suitable to prevent omission for user.

Reply:
Done, add default value 1.8.6.